Makefile

```diff
@@ -0,0 +1,65 @@
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.ONESHELL:
+.DELETE_ON_ERROR:
+MAKEFLAGS += --warn-undefined-variables
+MAKEFLAGS += --no-builtin-rules
+
+ifeq ($(origin .RECIPEPREFIX), undefined)
+  $(error This Make does not support .RECIPEPREFIX. Please use GNU Make 4.0 or later. Perhaps you meant gmake?)
+endif
+# .RECIPEPREFIX = >
+
+
+test: # run tests
+	@pwd
+	nvim --headless --noplugin -u tests/runner_init.vim -c "PlenaryBustedDirectory tests/malleatus/ {minimal_init = 'tests/test_init.vim'}"
+
+.PHONY: install help test
+
+
+.DEFAULT_GOAL := help
+# check https://stackoverflow.com/questions/4842424/list-of-ansi-color-escape-sequences
+NC = \033[0m
+ERR = \033[31;1m
+TAB := '%-20s' # Increase if you have long commands
+
+# tput colors
+red := $(shell tput setaf 1)
+green := $(shell tput setaf 2)
+yellow := $(shell tput setaf 3)
+blue := $(shell tput setaf 4)
+cyan := $(shell tput setaf 6)
+cyan80 := $(shell tput setaf 86)
+grey500 := $(shell tput setaf 244)
+grey300 := $(shell tput setaf 240)
+bold := $(shell tput bold)
+underline := $(shell tput smul)
+reset := $(shell tput sgr0)
```

Comment:
this (`tput`) is apparently CI-unfriendly but we can probably find some straightforward "if not interactive bail" make snippet